### PR TITLE
Fix CodeFactor brace style issues in FileSystemProvider

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
Fix instances of issue `The body of the else statement must be wrapped in opening and closing curly brackets.`

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [x] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/System.Management.Automation/namespaces/FileSystemProvider.cs [hunk 1]
```diff
@@ -964,8 +966,9 @@ protected override Collection<PSDriveInfo> InitializeDefaultDrives()
                             }
                         }
 
-                        if (skipDuplicate)
+                        if (skipDuplicate) {
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/FileSystemProvider.cs]: Please move brace to new line. Below too.
```suggestion
                        if (skipDuplicate)
                        {
```

File: src/System.Management.Automation/namespaces/FileSystemProvider.cs [hunk 2]
```diff
@@ -2126,10 +2141,12 @@ public static string Mode(PSObject instance)
             else if (itemType == ItemType.SymbolicLink || itemType == ItemType.HardLink)
             {
                 string action = null;
-                if (itemType == ItemType.SymbolicLink)
+                if (itemType == ItemType.SymbolicLink) {
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/FileSystemProvider.cs]: The same.

File: src/System.Management.Automation/namespaces/FileSystemProvider.cs [hunk 3]
```diff
@@ -2126,10 +2141,12 @@ public static string Mode(PSObject instance)
...
                     action = FileSystemProviderStrings.NewItemActionSymbolicLink;
-                else if (itemType == ItemType.HardLink)
+                }
+                else if (itemType == ItemType.HardLink) {
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/FileSystemProvider.cs]: The same.

File: src/System.Management.Automation/namespaces/FileSystemProvider.cs [hunk 4]
```diff
@@ -3936,7 +3961,9 @@ private void InitializeFunctionPSCopyFileFromRemoteSession(System.Management.Aut
 
         private void RemoveFunctionsPSCopyFileFromRemoteSession(System.Management.Automation.PowerShell ps)
         {
-            if ((ps == null) || !ValidRemoteSessionForScripting(ps.Runspace)) { return; }
+            if ((ps == null) || !ValidRemoteSessionForScripting(ps.Runspace)) {
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/FileSystemProvider.cs]: The same.

File: src/System.Management.Automation/namespaces/FileSystemProvider.cs [hunk 5]
```diff
@@ -3928,15 +3955,20 @@ private ArrayList GetRemoteSourceAlternateStreams(System.Management.Automation.P
 
         private void InitializeFunctionPSCopyFileFromRemoteSession(System.Management.Automation.PowerShell ps)
         {
-            if ((ps == null) || !ValidRemoteSessionForScripting(ps.Runspace)) { return; }
+            if ((ps == null) || !ValidRemoteSessionForScripting(ps.Runspace)) {
```
Comment (iSazonov) [inline src/System.Management.Automation/namespaces/FileSystemProvider.cs]: The same.

Comment (RDIL) [reply src/System.Management.Automation/namespaces/FileSystemProvider.cs]: Fixed

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
@RDIL Please rebase to pass CI-windows.

**Comment by iSazonov** [comment]
Please rebase to pass CIs.

**Comment by RDIL** [comment]
Done.  Sorry, was sleeping before.

**Comment by iSazonov** [comment]
@RDIL Thanks for your contribution!

Some notes for better productivity:
- please review your changes before push - it saves reviewer time.
- fix one style issue:
    - or in all code base (one commit for file or some files - we'll review commit by commit)
    - or by module. It is preferred. I suggest start with Utility module (any changes in Engine is dangerous!)
    - use automation if it possible (script or editor features) to reduce typos.

**Comment by RDIL** [comment]
Okay.

**Pull request merged**